pkgs/development/compilers/circt/default.nix

```diff
@@ -25,17 +29,15 @@ stdenv.mkDerivation rec {
 
   requiredSystemFeatures = [ "big-parallel" ];
 
-  nativeBuildInputs = [ cmake ninja git pythonEnv ];
+  nativeBuildInputs = [ cmake ninja git pythonEnv autoPatchelfHook ];
```

Comment:
Is `autoPatchelfHook` required? These libs has `$origin/../lib` in their rpath by default.

Reply:
No, it was not required. I'll remove it.